lib/instrumentation/mongodb/common.js

```diff
@@ -123,7 +123,7 @@ common.makeBulkDescFunc = function makeBulkDescFunc(shim) {
  *
  * @param {Shim} shim instance of shim
  * @param {object} instrumenter instance of mongo APM class
- * @param {object} [options={}] provide command names to skip updating host/port as they are unrelated to the active query.  This is only in v3 because after every command is runs `endSessions` which runs on the admin database
```

Comment:
Not sure why this change happened here. I tried to revert it but it's not working.

Reply:
can't you just copy what was there previously and recommit in that case?

Reply:
Just tried that and that fixed it. I did that before and it said no changes. Not sure what happened. But it's fixed now.